docs/docs/administration-and-security/access-control/rbac.md

```diff
@@ -26,30 +26,48 @@ left and open the **Users and Permissions** tab.
 
 ## Core concepts
 
-The diagram below shows an overview of how permissions are assigned within your Flagsmith organisation:
+### How permissions are assigned
 
-<div style={{textAlign: 'center'}}>
-```mermaid
-graph LR;
-    R[Custom roles] -->|Assigned to| G[Groups];
-    B[Built-in roles] -->|Assigned to| U[Users];
-    R -->|Assigned to| U;
-    R -->|Assigned to| A[Admin API keys];
-    G -->|Contains many| U;
-```
+Permissions are granted to **roles**, and roles are assigned to users, groups, or Admin API keys. A user's effective permissions are the **union of all permissions** from every role assigned to them — both directly and through group membership.
```

Comment:
Minor suggestion - should we add links to relevant documentation sections for `groups` and `Admin API Keys` (and users if applicable, but I'm not sure we have any documentation on users as such)?